src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -1212,6 +1212,60 @@ const schema = {
         },
       },
     },
+    secondaryAttachments: {
```

Comment:
Do we need this here anymore since it's in `vets-json-schema` ?

Reply:
no we don't need it, but i thought the schema in config/schema needed to match what was in vets-json-schema.